tests/kernel/stack/stack_api/src/test_stack_contexts.c

```diff
@@ -6,12 +6,13 @@
 
 #include <ztest.h>
 #include <irq_offload.h>
-
+#include <syscall_handler.h>
 #define STACK_SIZE 512
 #define STACK_LEN 2
 
 /**TESTPOINT: init via K_STACK_DEFINE*/
 K_STACK_DEFINE(kstack, STACK_LEN);
+K_STACK_DEFINE(kstack1, STACK_LEN);
```

Comment:
Please provide a more descriptive name

Reply:
@andrewboie changed the name kstack1 -> kstack_test_alloc